src/components/select/select.spec.js

```diff
@@ -1087,6 +1087,37 @@ describe('<md-select>', function() {
         expect($rootScope.testForm.$pristine).toBe(true);
       });
 
+        it('should keep the form dirty form is already dirty', function() {
```

Comment:
I think that "should not change a dirty form to pristine" is more clear.

Reply:
ok